plugins/linux-v4l2/v4l2-codec.h

```diff
@@ -26,42 +26,43 @@ extern "C" {
 #include <libavutil/pixfmt.h>
 
 /**
- * Data structure for mjpeg decoding
+ * Data structure for codec decoding
```

Comment:
"codec decoding" is sort of redundant ("coder decoder decoding").

Perhaps just "decoding"?